### DISCUSSION TITLE
G6里面如何实现这种效果

### DISCUSSION THREAD

**Discussion by ChenKun1997** [antvis/G6#3610] [Q&A]
G6里面如何实现下图这样，节点点击可以展开和收缩内容的效果，自定义节点一般都是固定的，看了图表示例也没发现这种可以通过点击事件实现部分shape显示隐藏的效果
![image](https://user-images.githubusercontent.com/24249578/161905162-defbcf7b-4a22-42b2-b8d1-a0f5cfc8c4ca.png)

**Comment by Yanyan-Wang** [comment]
示例中有局部事件的 https://g6.antv.vision/zh/examples/item/label#copyLabel。自定义节点里面也可以根据数据中的属性控制图形是否显示
